README.md

```diff
@@ -13,7 +13,7 @@ both [the installer](https://github.com/openshift/installer/) as well as Red Hat
 CoreOS. See also [the machine-api-operator](https://github.com/openshift/machine-api-operator)
 which handles provisioning of new machines - once the machine-api-operator
 provisions a machine (with a "pristine" base Red hat CoreOS), the MCO will take
-care of configuring it.
+care of serving the configuration.
```

Comment:
Changed this cause it's really misleading to think that we play an active role in the very first configuration of a new node. We just serve the generated machineconfigs via the mcs (but those were generated already on our side, bootstrap or running cluster). There's no active role we're playing here.

Reply:
The base configuration that turns into the Ignition that configures the system does live in this operator though...

Reply:
yea but the sentence says that "provision a machine and MCO will take care of configuring" which is not the case... the operator bootstrap for instance, already took care of rendering the ignitions and serving them through the mcs by the time the machines provision

Reply:
what you say is also true of course, but if we want to explain that we need to further fix the whole paragraph

Reply:
I guess when I write "MCO" I'm usually referring to "all of the components" including the MCS.  

The other thing here is that not only do we provide the Ignition on boot, we take care of configuration *after* it's booted the first time too.  The "configuring it" here covers both those cases.

Reply:
(BTW if you revise this PR can you also `s/Red hat/Red Hat/` above this?)

Reply:
> The other thing here is that not only do we provide the Ignition on boot, we take care of configuration _after_ it's booted the first time too. The "configuring it" here covers both those cases.

yes, the thing is, explaining that with just "The MCO takes care of configuring it" and talking about machine-api-operator in the same sentence is just too simplistic and have already created confusion on how all this works (had a lengthy discussion yesterday about this sentence,that's why I wanted to clarify but I guess that requires a full rewrite of this paragraph)
I'm reverting this for now, corrected RH typo instead